testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/iceberg/TestIcebergParquetFaultTolerantExecutionConnectorTest.java

```diff
@@ -68,11 +69,52 @@ public void testStatsBasedRepartitionDataOnInsert()
         throw new SkipException("We always get 3 partitions with FTE");
     }
 
+    @Override
+    @Test
+    public void testSortedNationTable()
+    {
+        throw new SkipException("File sorting for parquet files is handled by TestIcebergParquetConnectorTest");
```

Comment:
> FTE shouldn't result in much smaller files than non-FTE run, since tables with large number of (small) files are expensive to query. In an ideal world there shouldn't be such limitation

This should not be the case. Are you observing it? Actually I run the test with FTE and I got just one file and it passed just fine.

> it would be good to test Iceberg write-time sorting with FTE. For example, if we implemented sorting using SortNode in the plan, non-FTE would get data sorted and FTE wouldn't (am i right here @losipiuk @arhimondr )

If you would depend on the global sorting mechanism in Trino - then yeah - it would not work out of the box with FTE because of [this](https://github.com/trinodb/trino/blob/920d14c82c5adefd2f06feadfac1a0a96f9ff5ac/core/trino-main/src/main/java/io/trino/SystemSessionProperties.java#L1198-L1201)